code/modules/uplink/uplink_item.dm

```diff
@@ -1063,10 +1065,11 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	name = "Syndicate Surplus Crate"
 	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortion of items, \
 			but you never know. Contents are sorted to always be worth 50 TC."
+	item = /obj/structure/closet/crate
```

Comment:
It just needs a placeholder item in order to spawn, the actual `spawn_item` proc will ignore this and create the real item.